src/components/reactTasks/comment.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+export default class Comment extends React.Component{
+
+  constructor(){
+    super()
+    this.state = {comments: ['Brilliant!', 'Terrible'], value: ''}
+  }
+
+  handleChange(value) {
+    console.log(value)
```

Comment:
We always have to remember to take console.log()s out of the code before pushing to github. They're just for debugging.